### PR TITLE
Vonk: roadmap & patch

### PR DESCRIPTION
**Pull request by NJoyiT**

### CODE REVIEW

File: firelyserver/releasenotes/roadmap.rst [hunk 1]
```diff
@@ -7,52 +7,64 @@ This page lists the features and changes we have planned for the foreseeable fut
...
+
+* Full ONC/CMS compliance
+
+* Firely Server: ONC Edition
+
+   * Downloadable version of Firely Server that is fully compliant with the ONC Cures Act Final Rule
+
+* CAR factory/files
+
+   * Greatly reduces start-up time of Firely Server and other services
```
Comment (cknaap) [inline firelyserver/releasenotes/roadmap.rst:24]: It *will*, but we won't have it as a basis for Firely Server in Q1.

File: firelyserver/releasenotes/roadmap.rst [hunk 2]
```diff
@@ -7,52 +7,64 @@ This page lists the features and changes we have planned for the foreseeable fut
...
+
+   * Separate validation service optimized for usage in cloud environments
 
-* R4 search features
+* Firely Server: CMS Edition
 
-   * Search on reference by identifier
+   * Downloadable version of Firely Server that is fully compliant with the CMS Final Rule
+
+* Full compliance with MedMij standard
```
Comment (cknaap) [inline firelyserver/releasenotes/roadmap.rst:49]: Given the first item in Q3 I think you meant to remove it here.

File: firelyserver/releasenotes/roadmap.rst [hunk 3]
```diff
@@ -7,52 +7,64 @@ This page lists the features and changes we have planned for the foreseeable fut
...
-* R4 search features
+* Firely Server: MedMij Edition
 
-   * exponential numbers and quantities
-   * identifier:oftype modifier
+   * Downloadable version of Firely Server that is fully compliant with the MedMij standards
 
-* Async operations
-* Bulk data export
+* Bulk data export as a service
```
Comment (cknaap) [inline firelyserver/releasenotes/roadmap.rst:58]:
```suggestion
* Bulk data export as a separate service - deploy and scale it independently of Firely Server itself.
```

**Review by cknaap — CHANGES_REQUESTED** [review]

File: firelyserver/releasenotes/roadmap.rst (hunk 2 above)
Comment (NJoyiT) [reply firelyserver/releasenotes/roadmap.rst:49]: No, it are two different things: the one is actually being compliant and the other is offering a download with all the IGs preloaded as a solution, like we will for the ONC soon.

File: firelyserver/releasenotes/roadmap.rst (hunk 1 above)
Comment (NJoyiT) [reply firelyserver/releasenotes/roadmap.rst:24]: I know, but I'd still like to leave it as is. It will be updated when we have a clearer planning.

**Pull request merged**